bin/scripts/README.md

```diff
@@ -0,0 +1,59 @@
+# Utility Scripts for Candlepin
+
+### Refresh Data Extractor
+
+refresh_data_extractor.sh
```

Comment:
Suggestion: Maybe we can include the script name in the heading? This is more of a stylistic suggestion, I just want to see what you think and I think it can also work as you currently have it!

## Refresh Data Extractor - refresh_data_extractor.sh